includes/class-assets.php

```diff
@@ -525,7 +525,14 @@ public function enqueue_admin_scripts( $force = false ) {
 	 */
 	public function add_body_class( $admin_body_classes ) {
 		if ( WPBDP_App_Helper::is_bd_page() ) {
-			$admin_body_classes = ' wpbdp-admin-page';
+			$admin_body_classes .= ' wpbdp-admin-page';
+
+			// Hide expiration/renewal options, when renewal is OFF.
+			if( ! wpbdp_get_option( 'listing-renewal' ) ){
+				$admin_body_classes .= ' no-renewal';
```

Comment:
This class isn't being used in the css. The added css is causing the options to always be hidden.

Reply:
Thanks for the fix! It would also be great to include our prefix on the class.
`wpbdp-no-renewal`